sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -1676,6 +1677,19 @@ export class ApplicationService extends RecordDataModelService<Application> {
       );
     }
 
+    // Validates if the Program is active.
+    const program =
+      await this.educationProgramService.getActiveEducationProgram(
+        offering.educationProgram.id,
```

Comment:
Program `isActive` is already returned in `offering.educationProgram` 

![image](https://github.com/bcgov/SIMS/assets/54600590/8c784106-5a67-4ec9-8c38-f7a414940170)